core-shaded/pom.xml

```diff
@@ -146,7 +146,6 @@
                   -->
                   <include>com.datastax.oss:java-driver-core</include>
                   <include>io.netty:*</include>
-                  <include>com.esri.geometry:*</include>
```

Comment:
I'm a bit hesitant about this. Making a dependency optional and shading it are different concerns.

Initially, the decision to shade a dependency was taken by looking at the likelihood of a version conflict for a given dependency. For Esri, we deemed it high because the driver requires a very old Esri version. 

I think we should keep shading Esri even if it is optional, wdyt? This means that in the shaded jar Esri would be present and shaded, just like it is now. I can't see a situation where this could be a problem for the end user.

Reply:
I see your point here... I'm just not sure I agree.

To be clear, the case you're describing is fairly precise.  It would have to be something like the following:

* The users app uses ESRI 2.x for some other functionality
* The users app also uses the Java driver
* The user runs this app against DSE
* The user wants to leverage the geometric type support in DSE

This user will (possibly, even likely) experience a class conflict.  But the cost for supporting this (quite limited case) is that _every user of the shaded JAR_ has a larger JAR on their classpath.  Initially it seemed like optimizing for the broad case made sense rather than doing so for the extremely precise case but I'm certainly open to argument.

Reply:
> every user of the shaded JAR has a larger JAR on their classpath

Well the shaded jar already has all of Netty in it, so adding Esri on top of it won't really make a noticeable size difference :-)

But OK, I don't mind if we stop shading Esri.